gateway/Automation/Automation.hpp

```diff
@@ -82,6 +82,10 @@ void kitchenOff() {
   setOutput(KITCHEN_TABLE_ID, Relay::OFF);
 }
 
+void doorbellclick() {
+  dzwonek=1;
```

Comment:
For enumerated values Enum can be proposed based on CPP code style guidelines:
‘’’
enum DoorBellState
{
    DISABLED = 0,
    ENABLED = 1,
    SWITCHED = 2,
}
‘’’
See ‘Relay’ enum.